src/test/java/seedu/duke/command/FindCommandTest.java

```diff
@@ -15,46 +17,48 @@
 class FindCommandTest {
 
     Notebook notebook;
+    TagManager tagManager;
+    int maxRowLength = 100;
+
+    ArrayList<Tag> tag = new ArrayList<>();
+    Tag tagImpt;
 
     @BeforeEach
     void setup() {
         notebook = new Notebook();
+        tagManager = new TagManager();
+
         ArrayList<String> content = new ArrayList<>();
+
         content.add("default");
 
-        Note defaultNote = new Note("Default", content, false, false);
-        Note testNote1 = new Note("TestNote1", content, false, false);
+        tag.add(tagImpt);
+
+        Note defaultNote = new Note("Default", content, false, false, tag);
+        Note testNote1 = new Note("TestNote1", content, true, false);
         Note testNote2 = new Note("TestNote2", content, false, false);
+        Note testNote3 = new Note("Song Lyrics", content, true, false);
 
         notebook.addNote(defaultNote);
         notebook.addNote(testNote1);
         notebook.addNote(testNote2);
+        notebook.addNote(testNote3);
     }
 
     @Test
     void execute_keywordTest_returnsTestNote1AndTestNote2() {
-        String keyword = "Test";
-
-        String expected = "Here are the matching notes in your list:"
-                + Formatter.LS
-                + "1. TestNote1 "
-                + Formatter.LS
-                + "2. TestNote2 "
-                + Formatter.LS;
-        String result = getCommandExecutionString(notebook, keyword);
-
-        assertEquals(expected, result);
-
-    }
-
-    @Test
-    void execute_keywordDef_returnsDefault() {
-        String keyword = "def";
-
-        String expected = "Here are the matching notes in your list:"
-                + Formatter.LS
-                + "1. Default "
-                + Formatter.LS;
+        String keyword = "test";
+
+        String expected = "=".repeat(maxRowLength)
+                + FormatterStub.encloseRow(FindCommandStub.execute(keyword))
+                + "=".repeat(maxRowLength) + Formatter.LS
+                + "[96m1. Title: TestNote1 [0m"
+                + Formatter.LS + "default..." + Formatter.LS
+                + "=".repeat(maxRowLength) + Formatter.LS
+                + "[96m2. Title: TestNote2 [0m"
+                + Formatter.LS + "default..." + Formatter.LS
+                + "=".repeat(maxRowLength) + Formatter.LS
+                + "=".repeat(maxRowLength) + Formatter.LS;
```

Comment:
You may want to have an arraylist of note for the expected output and then use the Formatter.formatNotes. In the event that the format changes, you do not have to manually change the expected string again.